magefile.go

```diff
@@ -648,7 +650,7 @@ func (t Test) IntegrationCIPrepare(ctx context.Context) error {
 		return fmt.Errorf("creating cluster client: %w", err)
 	}
 
-	ctx = dev.ContextWithLogger(ctx, logger)
+	ctx = logr.NewContext(ctx, logger)
```

Comment:
Is there a reason why these changes are made?

Reply:
Not a major enough reason. Just something I noticed which was removed in the more recent versions of dev-kube, so attempted to comply the existing magefile with it.

Ref: https://github.com/mt-sre/devkube/commit/ed32c2091148d4de3c00a41d6b3450c01994aa24